### PR TITLE
 Fixed CORE-2576 for remove event log entry in the authentication filter

### PR DESCRIPTION
**Pull request by yamunaBQ**

### CODE REVIEW

File: rest-v2-app/src/main/java/org/ednovo/gooru/web/spring/interceptor/GooruInterceptor.java [hunk 1]
```diff
@@ -57,6 +58,12 @@
 	private static final Logger ACTIVITY_LOGGER = LoggerFactory.getLogger("activityLog");
 	
 	private static final JSONSerializer SERIALIZER = new JSONSerializer();
+
+	private static final String SESSIONTOKEN = "sessionToken";
+
+	private static final String _ORGANIZATION_UID = "organizationUid";
```
Comment (parthibane) [inline rest-v2-app/src/main/java/org/ednovo/gooru/web/spring/interceptor/GooruInterceptor.java]: @yamunaBQ  Please remove this constants if it's not used anywhere.

Comment (yamunaBQ) [inline rest-v2-app/src/main/java/org/ednovo/gooru/web/spring/interceptor/GooruInterceptor.java]: Its used here.

File: rest-v2-app/src/main/java/org/ednovo/gooru/web/spring/interceptor/GooruInterceptor.java [hunk 2]
```diff
@@ -31,6 +31,7 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import org.apache.cassandra.cli.CliParser.newColumnFamily_return;
```
Comment (parthibane) [inline rest-v2-app/src/main/java/org/ednovo/gooru/web/spring/interceptor/GooruInterceptor.java]: @yamunaBQ  what is the use of this import? it's not used in this class please remove.

Comment (yamunaBQ) [inline rest-v2-app/src/main/java/org/ednovo/gooru/web/spring/interceptor/GooruInterceptor.java]: Removed unused import.

**Pull request merged**